### PR TITLE
[RAC-6341]Modified UCS descovery graph to support ipmi pollers

### PR DESCRIPTION
**Pull request by mcgG**
**Background**
----
Additional value add feature which align with current RackHD ipmi poller feature; Second option besides original UCS poller.
The feasibility has been proved in previous experiment.
[https://rackhd.atlassian.net/browse/RAC-6341](https://rackhd.atlassian.net/browse/RAC-6341)

**AC**:
- Update discovery workflow and create poller

**Detail**
----
Added default ipmi poller for ucs, also user can config to skip the creation

jenkins depends on: https://github.com/RackHD/on-tasks/pull/534
Adding comment for concourse
depends on: https://github.com/RackHD/on-tasks/pull/534

@iceiilin @anhou @pengz1 @bbcyyb

### CODE REVIEW

File: lib/services/profile-api-service.js [hunk 1]
```diff
@@ -162,6 +162,7 @@ function profileApiServiceFactory(
                 skipReboot = 'false';
             }
             var skipPollers = configFile.get('skipPollersCreation', 'false');
+            var skipIpmiPollers = configFile.get('skipIPMIPollersCreation', 'false');
```
Comment (pengz1) [inline lib/services/profile-api-service.js]: It is not necessary to let UCS IPMI to be configured in RackHD configuration file.
You can use an 'and' operation to control this: 
if skipPollersCreation=false and skipIPMIPollersCreation=false, than create IPMI poller.
skipIPMIPollersCreation can be controlled by workflow not configuration file.

File: lib/graphs/ucs-discovery-graph.js [hunk 2]
```diff
@@ -173,6 +177,82 @@ module.exports = {
...
+                    pollers: [
+                        {
+                            'type': 'ipmi',
+                            'pollInterval': 60000,
+                            'config': {
+                                'command': 'sdr'
+                            }
+                        },
+                        {
+                            'type': 'ipmi',
```
Comment (anhou) [inline lib/graphs/ucs-discovery-graph.js]: Is this still valid for ucs pollers?

File: lib/graphs/ucs-discovery-graph.js [hunk 3]
```diff
@@ -173,6 +177,82 @@ module.exports = {
...
+                    ]
+                }
+            },
+            waitOn: {
+                'ucs-catalog': 'succeeded',
+                'skip-ipmi-pollers': 'failed'
+            }
+        },
+        {
+            label: 'noop-2',
```
Comment (anhou) [inline lib/graphs/ucs-discovery-graph.js]: What's the purpose of this task?

File: lib/graphs/ucs-discovery-graph.js (hunk 2 above)
Comment (pengz1) [reply lib/graphs/ucs-discovery-graph.js]: IPMI poller can be an enhancement of UCS poller.

File: spec/lib/services/profile-api-service-spec.js [hunk 4]
```diff
@@ -255,6 +255,7 @@ describe("Http.Services.Api.Profiles", function () {
         configuration.get.withArgs('skipResetPostDiscovery').returns('false');
         configuration.get.withArgs('autoCreateObm').returns('false');
         configuration.get.withArgs('skipPollersCreation').returns('false');
+        configuration.get.withArgs('skipIPMIPollersCreation').returns('false');
```
Comment (pengz1) [inline spec/lib/services/profile-api-service-spec.js]: Remove this line.

**Pull request merged**